recipe/conda_build_config.yaml

```diff
@@ -20,22 +20,22 @@ cxx_compiler:
   - clangxx                    # [osx]
   - vs2017                     # [win]
 cxx_compiler_version:          # [unix]
-  - 11                         # [osx]
-  - 9                          # [linux]
+  - 12                         # [osx]
+  - 10                         # [linux]
   - 7                          # [os.environ.get("CF_CUDA_ENABLED", "False") == "True" and linux64]
   - 9                          # [os.environ.get("CF_CUDA_ENABLED", "False") == "True" and linux64]
-  - 9                          # [os.environ.get("CF_CUDA_ENABLED", "False") == "True" and linux64]
-  - 9                          # [os.environ.get("CF_CUDA_ENABLED", "False") == "True" and linux64]
+  - 10                         # [os.environ.get("CF_CUDA_ENABLED", "False") == "True" and linux64]
```

Comment:
Should the fortran version also be bumped to 10 for these CUDA versions?

Reply:
`c_compiler_version` also needs to be fixed.

Reply:
I fixed it, but fortran + CUDA 10.2 already wouldn't work because CUDA 10.2 requires fortran 7 which is incompatible with the rest of conda-forge.

Reply:
At least it is consistently broken then :D